docker/star/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:10 AS python
+FROM debian:12.5-slim AS python
```

Comment:
Has this change been validated in `pytest`?

Reply:
I didn't do anything special, but the `pytest` check was passing.

Reply:
I don't see in the git history any tests that were pointing to this branch's built image

Reply:
Good point. Now I'm wondering if the `pytest` check should automatically change the container entries to use the branch images on PR tests.

Reply:
That might solve another problem we have as well! I think that would let the "last commit" in a PR pass the check (issue outlined in #189 )